subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -88,6 +97,17 @@ static int get_published_payload(struct mqtt_client *client, u8_t *write_buf,
 	return 0;
 }
 
+/**
+ * @brief Wait lock used in update_job_execution to wait for when the previous
```

Comment:
Mentioning who the caller of this function is can quickly become stale information :D.

Something like "Wait for a job update to be accepted." is plenty good enough IMO.

Reply:
Okei I'll add it